code_of_conduct.md

```diff
@@ -0,0 +1,37 @@
+# Code of Conduct
+
+The Women Who Code DC Mission has three main goals:
+1. That women software engineers feel confident and empowered to pursue their goals.
+1. That every woman codes
+1. That all women who code around Washington DC have a community they can access for support and resources.
+
+It is our responsibility as an organization to cultivate a safer environment for all women (and nonbinary people, too). We are committed to being welcoming and embracing of people of all ages, races, ethnicities, nationalities, sexualities, gender expressions, disabilities, religions, skill levels.
+
+This code of conduct applies for all official Women Who Code DC functions, including conversations online.
+
+### No Harrassment Policy
+
+Harassment is any unwelcome or hostile behavior towards another person for any reason. This includes, but is not limited to, offensive verbal comments related to personal characteristics or choices, sexual images or comments, deliberate intimidation, bullying, stalking, following, harassing photography or recording, sustained disruption of discussion or events, nonconsensual publication of private comments, inappropriate physical contact, or unwelcome sexual attention. Conduct need not be intentional to be harassment.
+
+Women Who Code DC will not tolerate such behavior and will make efforts to address these issues sensitively and justly.
+
+### Making a Report
+
+If you see or experience someone violating this code of conduct and would like support from the leadership, you can contact any member of our leadership in person or electronically. Leads will keep these conversations private and take action only with consent of the affected member. Contact information about all active leads can be found at [THIS NEEDS TO POINT SOMEWHERE]().
```

Comment:
is there a email alias for just the leads? what is the current email/contact paths that exist currently for WWHC ?